.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+**Description**
+
+Please include a summary of the change and which issue is fixed. Include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue) (if applicable)
+
+**Type of change**
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected, and would require a major version change)
```

Comment:
Should we expose "and would require a major version change" here ? It's really not up to a contributor whether this is required and this text being here might make it seem like they should consider versioning when they really don't need to, they just need to consider whether it changes existing behavior.

Reply:
Hmm, well my reasoning for it is that the reviewer/merger who is reading the PR, would be reminded that merging the breaking change would require the next release be a major one, instead of merging and forgetting. Though I suppose this could still lead to merging and forgetting anyway so I guess it doesn't matter. I'll probably remove it then.

Reply:
If we really want to prevent mistakes, we should really have a separate checklist specifically for merging that the merger absolutely has to go through no matter what before merging.

Reply:
I looked into this briefly, it is technically possible with some CI to:
- create merger's checklist when PR is opened (as having it in the original template would allow the person who opened it to edit the checklist (unless they are admin on the repo anyway))
- prevent merge until all items in the merger's checklist is complete

but its hella complex and not necessary